cnf-certification-test/accesscontrol/suite.go

```diff
@@ -32,111 +32,119 @@ import (
 	"github.com/test-network-function/cnf-certification-test/pkg/provider"
 	"github.com/test-network-function/cnf-certification-test/pkg/testhelper"
 	"github.com/test-network-function/cnf-certification-test/pkg/tnf"
+	"github.com/test-network-function/test-network-function-claim/pkg/claim"
 )
 
+type Test struct {
+	id claim.Identifier
+	do func(*provider.TestEnvironment)
+}
+
 var (
-	nonCompliantCapabilities = []string{"NET_ADMIN", "SYS_ADMIN", "NET_RAW", "IPC_LOCK"}
+	nonCompliantCapabilities = []string{
+		"NET_ADMIN",
+		"SYS_ADMIN",
+		"NET_RAW",
+		"IPC_LOCK",
+	}
 	invalidNamespacePrefixes = []string{
 		"default",
 		"openshift-",
 		"istio-",
 		"aspenmesh-",
 	}
+	testContainer = []Test{
```

Comment:
I would change these new slice names to `containerTests`, `podTests` and `namespaceTests`.